modules/fs2/src/main/scala/higherkindness/mu/rpc/internal/client/fs2/calls.scala

```diff
@@ -86,72 +85,64 @@ object calls {
       }
       .pure[F]
 
-  def tracingClientStreaming[F[_]: Async, Req, Res](
-      input: Stream[Kleisli[F, Span[F], *], Req],
+  def contextClientStreaming[F[_]: Async, C, Req, Res](
+      input: Stream[Kleisli[F, C, *], Req],
       descriptor: MethodDescriptor[Req, Res],
       channel: Channel,
       options: CallOptions
-  ): Kleisli[F, Span[F], Res] =
-    Kleisli[F, Span[F], Res] { parentSpan =>
-      parentSpan.span(descriptor.getFullMethodName()).use { span =>
-        span.kernel.flatMap { kernel =>
-          val headers = tracingKernelToHeaders(kernel)
-          val streamF: Stream[F, Req] =
-            input.translate(Kleisli.applyK[F, Span[F]](span))
-          clientStreaming[F, Req, Res](
-            streamF,
-            descriptor,
-            channel,
-            options,
-            headers
-          )
-        }
+  )(implicit C: ClientContext[F, C]): Kleisli[F, C, Res] =
```

Comment:
_question_: would it be better if we rename this `C `implicit `val` differently from the `C[F]` implicit bound? (one or the other)

(same suggestion for the rest of the methods too)